styles/styles.css

```diff
@@ -0,0 +1,264 @@
+body {
+    font-family: 'Berkshire Swash', cursive;
+}
+p, nav {
+    font-family: 'Anaheim', sans-serif;
+}
+/*Header*/
+.header {
+    background-image: url(/images/promo-background.jpeg);
+    height: 420px;
+    width: 99.5%vw; 
+    color: white;
+}
+.top-container {
+    display: flex;
+}
+/*Logo*/
+.logo {
+    margin-top: 10px;
+    margin-left: 40px;
+    margin-right: -60px;
+}
+/*End of logo*/
+ul {
+    list-style-type: none;
+    display: inline-flex;
+    padding: 20px;
+    color: white;
+    font-family: 'Berkshire Swash', cursive;;
+    font-size: small;
+    margin-left: 240px;
+}
+ul a {
+    padding: 10px;
+    color: white;
+}
+ul a:hover {
+    background-color: darkgreen;
+    opacity: 30%;
+    color: white;
+}
+ul a.current {
+    background-color: darkred;
+    opacity: 35%;
+}
+ul a.current:hover {
+    opacity: 60%;
+}
+.header > h1 {
+    text-align: center;
+    font-size: 50px;
+
+}
+.header > p {
+    text-align: center;
+}
+.buttons {
+    text-align: center;
+}
+.btn-1 {
+    background-color: black;
+    height: 35px;
+    width: 140px;
+    opacity: 60%;
+    color: white;
+    border: none;
+}
+.btn-1:hover {
+    opacity: 100%;
+    border: white 1px solid;
+}
+.btn-2 {
+    background-color: white;
+    height: 35px;
+    width: 140px;
+    opacity: 30%;
+    color: black;
+    border: none;
+}
+.btn-2:hover {
+    opacity: 100%;
+    border: darkgrey 1px solid;
+}
+/*Cook and dish (Just the right food)*/
+.cook {
+    background-image: url(/images/about-background.jpeg);
+    height: 400px;
+    width: 99.5% vw;
+    display: flex;
+    text-align: center;
+    justify-content: center;
+}
+.cook h2 {
+    color: goldenrod;
+    margin-top: 45px;
+}
+.cook p {
+    width: 300px;
+    font-size: small;
+    margin-right: 20px;
+    margin-left: 20px;
+}
+#about-cook {
+    height: 130px;
+    width: 130px;
+}
+#about-cook:hover {
+    transform: scale(1.1);
+}
+#steak-plate {
+    height: 340px;
+    width: 340px;
+    margin-top: 30px;
+}
+#steak-plate:hover {
+    transform: scale(1.1);
+}
+/*Ingredients (Fine ingredients)*/
+.ingredients {
+    background-image: url(/images/ingredients-background.jpeg);
+    height: 400px;
+    width: 99.5%vw;
+    display: flex;
+}
+.container {
+    height: 320px;
+    width: 320px;
+    background-color: black;
+    opacity: 70%;
+    color: white;
+    margin-top: 36px;
+    margin-left: 450px;
+    font-size: small;
+    text-align: center;
+}
+.container:hover {
+    opacity: 85%;
+}
+.container p {
+    margin-left: 25px;
+    margin-right: 25px;
+}
+.small-container {
+    display: flex;
+    justify-content: space-evenly;
+}
+.miniatures {
+    height: 65px;
+    width: 65px;
+    opacity: 100%;
+    /*How can I get non-transparent pictures here?*/
```

Comment:
Ahah! So if I understand you right - you want to include a background image which is transparent. You would need a specific type of image, most commonly a `.png` file.